flows/playground.ts

```diff
@@ -479,20 +479,20 @@ export const playgroundFlows: Flow[] = [
     congratulationTexts: ["Congratulations you're an Active Aztec user!"],
   },
   {
-    path: "timeswap-lens-followers",
```

Comment:
@Sigri44, can you pull latest version from main ? You should add your own instead of modifying this one